test/e2e/operator_test.go

```diff
@@ -631,6 +631,96 @@ func TestInternalLoadBalancer(t *testing.T) {
 	}
 }
 
+// TestInternalLoadBalancerGlobalAccessGCP creates an ingresscontroller
+// with internal load balancer on GCP with the GCP Global Access provider
+// parameter set to both "Global" and "local" to verify that the
+// Load Balancer is service created properly.
```

Comment:
nit: Load Balancer service is created properly.

Reply:
fixed. thanks!